src/story_protocol_python_sdk/abi/SPGNFTImpl/SPGNFTImpl_client.py

```diff
@@ -0,0 +1,23 @@
+
+import json
+import os
+from web3 import Web3
+
+class SPGNFTImplClient:
+    def __init__(self, web3: Web3, contract_address=None):
```

Comment:
`init` will fail is `contract_address` is empty, so I think you need to add some verification, something like 

```
if contract_address is None:
            raise ValueError("Contract address is required")
```